AgentDiscoveries-Backend/src/main/resources/META-INF/persistence.xml

```diff
@@ -1,16 +1,16 @@
-<persistence xmlns="http://java.sun.com/xml/ns/persistence"
-             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
-             version="2.0">
-
-    <persistence-unit name="org.softwire.training.jpa">
-        <properties>
-            <property name="javax.persistence.jdbc.driver" value="com.mysql.jdbc.Driver" />
-            <property name="javax.persistence.jdbc.url" value="jdbc:mysql://localhost:3306/agentdiscoveries?useSSL=false" />
-            <property name="javax.persistence.jdbc.user" value="" />
-            <property name="javax.persistence.jdbc.password" value="" />
-        </properties>
-    </persistence-unit>
-
-</persistence>
-
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
```

Comment:
I'm not sure what has changed for this file.. might be to do with line breaks?

Reply:
Thinking more, this is likely to disrupt others, so let's try and undo this change.